example/netcdf/t8_write_forest_netcdf.cxx

```diff
@@ -13,6 +13,99 @@
 
 T8_EXTERN_C_BEGIN ();
 
+/* Function that times the duration of writing out the netCDF File, given a specific variable storage and access pattern */
+static void
+t8_example_time_netcdf_writing_operation (t8_forest_t forest,
+                                          sc_MPI_Comm comm,
+                                          int netcdf_var_storage_mode,
+                                          int netcdf_var_mpi_access,
+                                          const char *title)
+{
+#if T8_WITH_NETCDF
+  double              start_time, end_time, duration, global;
+  int                 retval;
+  sc_MPI_Barrier (comm);
+  start_time = sc_MPI_Wtime ();
```

Comment:
Note: In future we can do the time measurement via sc_statistics.
If you look for example into the t8_time_forest_partition example how its done.
The sc_statistics can compute min,max and average for all MPI ranks by itself.